src/const.js

```diff
@@ -109,6 +109,19 @@ export const IMAGE_FILE_EXTENSIONS = [
   'svg',
 ];
 
+// Map the image mime to the expected file extensions
+// (used in the the static theme images validation).
+export const IMAGE_MIMES_TO_EXTENSIONS = {
```

Comment:
I feel like this name could be worded a bit better, but it seems tough without making it really long. If you can come up with something better I wouldn't be opposed to it ;). I think MIME should be singular and EXTENSIONS might be too overloaded for add-ons code.

Maybe `IMG_MIME_TO_FILE_EXTS` or `MIME_TO_FILE_EXTENSIONS`?